Interpreter.py

```diff
@@ -40,7 +41,7 @@ def brainfuck(program):
             data_pointer -= 1
         elif command == '+':
             data[data_pointer] = (data.get(data_pointer, 0) + 1)
-            if data[data_pointer] == 256:
+            if data[data_pointer] == (int(255 / 8) * int(bits)) + 1:
```

Comment:
Please change this line to `2 ** bits`, it is much clearer than this calculation :)

Reply:
Forgot how bits work lol.
Uninstalling Windows out of rage...
Uninstalled Windows.
Throw computer out of window.
Threw computer out of window.
jk not really